roles/contiv_network/tasks/main.yml

```diff
@@ -107,6 +107,15 @@
   retries: 9
   delay: 10
 
+# get the netmaster status for debugging in case there are errors
+- name: get netmaster status
```

Comment:
@gaurav-dalvi, @unclejack, Sorry I missed this update. "shell netctl tenant ls" makes sure that netmaster is up and there is a default tenant. "systemctl status netmaster -l" gets the systemd service status and the logs for the service. In some cases when netmaster does not come up, the service may be down or sometimes the logs show what went wrong even if the service is active. So they serve different purposes, and this status is only for logging/debugging.